proxies/go/handler_command.go

```diff
@@ -4,14 +4,17 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
 
 	devcycle "github.com/devcyclehq/go-server-sdk"
 	"github.com/gorilla/mux"
 )
 
 type Param struct {
-	Type  *string `json:"type,omitempty"`
-	Value *any    `json:"value,omitempty"`
+	Type  string `json:"type,omitempty"`
```

Comment:
did this need to be changed? Using pointers here makes the "empty" state a nil value which I think is easier to check for

Reply:
good point, i think i accidentally changed it back fixing merge conflicts - fixed!